RunSamples.cmd

```diff
@@ -1,6 +1,23 @@
 @echo OFF
 setlocal
 
+set VERBOSE=
+
+:argsloop
+
+if "%1" == "" (
+    rem - no more arguments. 
+) else (
+    rem - check each argument
+    if "%1" == "--verbose" (
+        set VERBOSE="verbose"
+        @echo [RunSamples.cmd] VERBOSE is %VERBOSE%
```

Comment:
I think this line will always output `[RunSamples.cmd] VERBOSE is ` without `VERBOSE` due to environment variable expansion issue.